packages/react/src/_popover/Popover.tsx

```diff
@@ -112,6 +114,7 @@ export function usePopover<RT extends ReferenceType = ReferenceType>(
 							  }
 							: {
 									maxWidth: `${availableWidth}px`,
+									overflowX: 'auto',
```

Comment:
Just checking that this `overflowX` is required?

Reply:
Nah, this isn't needed, I'll remove this.